packages/chart/src/components/LinearChart.tsx

```diff
@@ -402,7 +398,7 @@ const LinearChart = forwardRef<SVGAElement, LinearChartProps>(({ parentHeight, p
 
     // Heights to add
 
-    const brushHeight = brush?.active ? brush?.height + brush?.height : 0
+    const brushHeight = config.brush?.active ? 50 : 0
```

Comment:
Assign 50 to a variable so we know why it was used in the future.

Reply:
Fixed